lib/omloga.rb

```diff
@@ -99,16 +109,18 @@ def get_pid(line)
   end
 
   File.foreach($LOG_FILE) do |log_line|
+    log_line.strip!
     pid = get_pid(log_line)
-    req = request_hash[pid]
+    uuid = get_uuid(log_line)
```

Comment:
Need to handle `uuid` being nil. If it is nil then we just skip that line and deal with it like we do below.